tests/lib/prepare.sh

```diff
@@ -214,6 +214,15 @@ update_core_snap_for_classic_reexec() {
     done
 }
 
+flush_changes() {
+    sleep 1
+    local timeout=20
+    while [ "${timeout}" -gt 0  ] && snap changes | grep -q Doing; do
```

Comment:
Silly question, could this be just: `retry -n 20 sh -c 'snap changes | grep -q Doing'` ?

Reply:
I am running the CI again. I will squash if that worked.

Reply:
Done.